src/devices/tuya.ts

```diff
@@ -773,13 +773,14 @@ const definitions: Definition[] = [
             e.binary('preheat', ea.STATE, true, false).withDescription('Indicates sensor preheat is active'),
         ],
         whiteLabel: [
-            tuya.whitelabel('TuYa', 'DY-RQ500A', 'Gas sensor', ['_TZE204_zougpkpy']),
+            tuya.whitelabel('DYGSM', 'DY-RQ500A', 'Gas sensor', ['_TZE204_zougpkpy']),
         ],
         meta: {
             tuyaDatapoints: [
-                [1, 'gas', tuya.valueConverter.trueFalseEnum0],
+                [1, 'gas_detection_state', tuya.valueConverterBasic.lookup({'alarm': tuya.enum(0), 'normal': tuya.enum(1)})],
```

Comment:
I propose to keep `[1, 'gas', tuya.valueConverter.trueFalseEnum0],`, it is easier to handle with `true`/false` than `alarm`/`normal`.

Reply:
But:
```
        e.enum('gas_detection_state', ea.STATE, [true, false]).withDescription('State of sensor'),

            [1, 'gas_detection_state', tuya.valueConverterBasic.lookup({True: tuya.enum(0), False: tuya.enum(1)})],

```
<img width="554" alt="Captura de Pantalla 2023-10-30 a las 8 37 05" src="https://github.com/Koenkk/zigbee-herdsman-converters/assets/5563436/2b9b3b21-40bd-460a-b9c5-4f92c802e926">

<img width="519" alt="Captura de Pantalla 2023-10-30 a las 8 40 10" src="https://github.com/Koenkk/zigbee-herdsman-converters/assets/5563436/3f7fe103-03f0-4131-be91-b8949d5921bd">